task/github.py

```diff
@@ -381,22 +381,15 @@ def allowlist(self):
         users = {"github-actions[bot]", "candlepin", "cockpit-project", "osbuild"}
 
         # individual persons from https://github.com/orgs/cockpit-project/teams/contributors/members
-        teamId = self.teamIdFromName(TEAM_CONTRIBUTORS)
```

Comment:
You can also remove then "TEAM_CONTRIBUTORS" constant.

Reply:
Thanks for spotting! Fixed.